src/lib/source-handlers/azure/list-repos.ts

```diff
@@ -0,0 +1,84 @@
+import { AzureRepoData } from './types';
+import Bottleneck from 'bottleneck';
+import base64 = require('base-64');
+import * as debugLib from 'debug';
+import { getAzureToken } from './get-azure-token';
+import { getAzureOrgName } from './get-azure-orgname';
+import * as needle from 'needle';
+
+const debug = debugLib('snyk:azure');
+const azureDefaultUrl = 'https://dev.azure.com';
```

Comment:
in app.snyk.io we support a custom azure repos url, aka self hosted so we need to support it the same way we do for Gitlab & Github Enterprise + confirm the API used is the same

<img width="1129" alt="CleanShot 2021-11-22 at 09 58 20@2x" src="https://user-images.githubusercontent.com/2911613/142840906-3901d9e9-6b2e-4ca2-ba17-5a2a090165ea.png">
e.

Reply:
Will apply

Reply:
Done